src/hashbang.sh

```diff
@@ -111,7 +111,9 @@ makekey() {
 
 # Fetch host data for later.
 # If this fails there is no point in proceeding
-host_data=$(curl -sH 'Accept:text/plain' https://hashbang.sh/server/stats)
```

Comment:
It is indeed for fixing the line break bug. However, it should also indeed be removed when the script aborts / fails, or when something else goes bad. And when we're done with it. I'll probably get to that eventually, unless anyone else fixes it first.